src/pages/ControllersIndex/ControllersIndex.tsx

```diff
@@ -155,26 +192,28 @@ function Details() {
             message={loginError}
```

Comment:
This is a small unrelated issue, but I was thinking it might still be good to fix it. When the text of the error is too long, we can't view it entirely on a small screen. I think that using `breakLines` (that we've added in #1631) and adding `tooltipClassName="p-tooltip--fixed-width"` as a prop to this `Tooltip` would fix the issue.

![Screenshot from 2023-09-19 03-07-38](https://github.com/canonical/juju-dashboard/assets/108150922/3ea15ccb-d1e0-4387-990d-0533219b2501)

Reply:
Done:

<img width="294" alt="Screenshot 2023-09-19 at 12 03 29 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/c7741dd3-bae7-41e9-84b6-a205a83e15d2">